example/spec/kinesisTests/KinesisTestTriggerSpec.js

```diff
@@ -0,0 +1,154 @@
+'use strict';
+
+const { s3 } = require('@cumulus/common/aws');
+
+jasmine.DEFAULT_TIMEOUT_INTERVAL = 550000;
+
+const {
+  LambdaStep,
+  waitForCompletedExecution
+} = require('@cumulus/integration-tests');
+const { randomString } = require('@cumulus/common/test-utils');
+
+const { loadConfig } = require('../helpers/testUtils');
+const {
+  createOrUseTestStream,
+  putRecordOnStream,
+  waitForActiveStream,
+  waitForTestSfStarted
+} = require('../helpers/kinesisHelpers');
+
+const record = require('../../data/records/L2_HR_PIXC_product_0001-of-4154.json');
+
+const granuleId = record.product.name;
+const recordIdentifier = randomString();
+record.identifier = recordIdentifier;
+
+const testConfig = loadConfig();
+
+const lambdaStep = new LambdaStep();
+const streamName = testConfig.streamName;
+
+
+const recordFile = record.product.files[0];
+const expectedTranslatePayload = {
+  granules: [
+    {
+      granuleId: record.product.name,
+      files: [
+        {
+          path: 'cumulus-test-data/pdrs',
+          url_path: recordFile.uri,
+          bucket: record.bucket,
+          name: recordFile.name,
+          size: recordFile.size
+        }
+      ]
+    }
+  ]
+};
+
+const fileData = expectedTranslatePayload.granules[0].files[0];
+const filePrefix = `file-staging/${testConfig.stackName}/L2_HR_PIXC`;
+
+const fileDataWithFilename = {
+  ...fileData,
+  filename: `s3://${testConfig.buckets.private.name}/${filePrefix}/${recordFile.name}`,
+  bucket: testConfig.buckets.private.name,
+  url_path: '',
+  fileStagingDir: filePrefix
+};
+
+const expectedSyncGranulesPayload = {
+  granules: [
+    {
+      granuleId: granuleId,
+      files: [fileDataWithFilename]
+    }
+  ]
+};
+
+
+// When Cumulus is configured to trigger a CNM workflow from a Kinesis stream and a message appears on the stream, Cumulus triggers the workflow
```

Comment:
It might be helpful to be more specific, e.g.:
> When kinesis-type rules exist, the Cumulus lambda `kinesisConsumer` is configured to trigger workflows when new records arrive on a Kinesis stream. When a record appears on the stream, the `kinesisConsumer` lambda triggers workflows associated with the kinesis-type rules.

Reply:
That's funny because I was going to delete that comment, I copied it from the jira ticket since patrick said to make our tests follow that language.

Reply:
but I will update it with your text.